internal/plugin.go

```diff
@@ -394,6 +395,8 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 			policy.ComplianceType = p.PolicyDefaults.ComplianceType
 		}
 
+		// No need to set policy.MetadataComplianceType here since default value is ""
```

Comment:
This isn't quite right since the user can set a default to this value, for example:
```
policyDefaults:
  metadataComplianceType: musthave
```

Reply:
I did it as current PR  because I'm not sure if we also want `metadataComplianceType` under `policyDefaults`, or just under `policies` and `manifests` as `complianceType`, since `metadataComplianceType` is an enhancement of `complianceType`. If confirmed, I can update it here.

Reply:
@ChunxiAlexLuo yes, but you added it to the `PolicyDefaults` structure, which makes it available under `policyDefaults` in the YAML.

Reply:
Also, just about every other option allows for inheritance from `policyDefaults` -> `policies`, so it'd be nice to be consistent here as well.